src/i18n/en-GB.js

```diff
@@ -461,6 +461,7 @@ module.exports = {
     "SANDBOX.EDIT.FILE":"Can modify file chosen by user",
     "SANDBOX.READ.FOLDER":"Can read selected files of the associated types from folder chosen by user",
     "SANDBOX.MESSAGES":"Can exchange messages with friends",
+    "SANDBOX.PROFILE.PHOTO":"Can retrieve profile photos shared with you",
```

Comment:
Can retrieve profile photos of your friends?

or maybe even more generic - "Read profiles of your friends"?